spyder/plugins/layout/plugin.py

```diff
@@ -532,7 +521,7 @@ def load_window_settings(self, prefix, default=False, section='main'):
         # with the current screen. See spyder-ide/spyder#3748.
         width = pos[0]
         height = pos[1]
-        screen_shape = self.main.screen().geometry()
+        screen_shape = self.main.screen().virtualGeometry()
```

Comment:
Maybe describing the advantage or need for using `virtualGeometry` instead of `geometry` here could be worthy